### PR TITLE
Adds link variation to button

### PR DESCRIPTION
**Pull request by froskie**
## What this PR do
- Adds link variation to button
image

## Where should the reviewer start:
- Diffs

### CODE REVIEW

File: components/button/button.scss [hunk 1]
```diff
@@ -99,3 +99,24 @@
   font-size: 28px;
   padding: 15px 20px;
 }
+
+.ui-button_variation_link {
+  display: inline-block;
+  background: transparent;
```
Comment (yurist38) [inline components/button/button.scss]: I guess, we have an agreement to sort alphabetically css props...

Comment (froskie) [inline components/button/button.scss]: You're right

File: themes/_default.yaml [hunk 2]
```diff
@@ -145,6 +145,21 @@ button:
     font-family: "var(--tx-generic-font-primary-font-family), var(--tx-generic-font-primary-generic-family)"
     hover-color: *primary
     hover-border-color: *primary
+  link:
+    color: *text
+    font:
+      family: "var(--tx-generic-font-secondary-font-family)"
+      size: "14px"
+      weight: "var(--tx-generic-font-primary-weight-regular)"
```
Comment (asci) [inline themes/_default.yaml]: can't you use these variables directly?

Comment (froskie) [inline themes/_default.yaml]: By directly you mean the actual values? The way it is adds more flexibility per theme.

**Pull request merged**